specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```diff
@@ -0,0 +1,1348 @@
+          "enum": [
+            "Scaling",
+            "Updating",
+            "Creating",
+            "Deleting",
+            "Succeeded",
+            "Failed"
+          ],
+          "x-ms-enum": {
+            "name": "InferenceEndpointDeploymentProvisioningState",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "The current provisioning state of the Inference Deployment resource."
+        },
+        "environmentConfigId": {
+          "type": "string",
+          "description": "ResourceId of the environment configuration Artifact.",
+          "readOnly": true
+        },
+        "environmentOverrides": {
+          "type": "object",
+          "description": "Environment variables that override any default environment variables in the container.",
+          "readOnly": true,
+          "properties": {
+            "environmentVariables": {
+              "description": "Contains environment variables defined as key/value pairs.",
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            "startupCommand": {
+              "description": "Command to start the container.",
+              "type": "string"
+            }
+          }
+        },
+        "codeAssetId": {
+          "description": "Resource Id of the Code Assets i.e. startup code.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "InferenceEndpointCreateRequest": {
+      "description": "The request to create an Inference Endpoint.",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The description of the endpoint.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Contains resource tags defined as key/value pairs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "InferenceEndpointDeploymentCreateRequest": {
+      "description": "The request to create an Inference Deployment.",
+      "type": "object",
+      "required": [
+        "modelIds",
+        "sku",
+        "weight"
+      ],
+      "properties": {
+        "tags": {
+          "description": "Contains resource tags defined as key/value pairs.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "modelIds": {
+          "description": "The array of model resource Ids.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "minItems": 1,
+          "maxItems": 10
+        },
+        "sku": {
+          "description": "Sku for the deployment. Give the name of the sku such as Standard_DS2.",
+          "type": "string",
+          "enum": [
```

Comment:
remove enum for now... we are yet to come up with SKU names

Reply:
Lets change to AmlInferenceD2v2, AmlInferenceD3v2

Reply:
Why dont we be explicit on what vm skus we are using here?